database/mssql/scripts/dbscripts/PSP_PIMS_S29_00/Alter Down/01_DML_PIMS_ROLE_CLAIM.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM PIMS_ROLE_CLAIM
```

Comment:
Start the alter down by removing all role claims, as we the alter down/up create them from scratch.